README.md

```diff
@@ -32,31 +32,418 @@ There's a very unstable version of the plugin available to try out. It's held to
 
 `yarn rw dev`
 
-## API Reference
+> **Note**
+>
+> The following steps won't be needed for much longer
 
-After setting up your app via the CLI, you'll find a demo page has been added to your app. This is intended as an in-app API reference, though in a proper store it might a little different. 
+5. Checkout a subscription item using "user@test.com" as the email.
 
-### `<StripeProvider/>`
+6. Configure Customer Portal in the [Stripe Dashboard](https://dashboard.stripe.com/subscriptions) 
 
-### `useStripeCheckout`
+7. Play around!
+
+## Web-side API Reference
+
+After setting up your app via the setup command, you'll see that a demo page has been added to your app. This is intended as an in-app API reference—I'm assuming you would want your commerce project to look a different.
```

Comment:
When writing these readme's try not to use first person. The project has several collaborators so it should reflect that.

Reply:
Good point!